src/components/ForecastSummaries.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "../styles/ForecastSummaries.css";
+import ForecastSummary from "./ForecastSummary";
+
+function ForecastSummaries({ forecasts, onForecastSelect }) {
+  return (
+    <div className="forecast-summaries">
+      {forecasts.map((forecast) => (
+        <ForecastSummary
+          key={forecast.date}
+          date={forecast.date}
+          description={forecast.description}
+          icon={forecast.icon.toString()}
```

Comment:
Instead of changing the data type inline like this, would it not be better to change it to the format you want at the source of the data (in this case in `getForecasts`)? You could make your proptypes uniform from here on out then